src/arch-x86_64-syscalls.c.perf

```diff
@@ -0,0 +1,565 @@
+%{
+/**
+ * Enhanced Seccomp x86_64 Syscall Table
+ *
+ * Copyright (c) 2012 Red Hat <pmoore@redhat.com>
+ * Author: Paul Moore <paul@paul-moore.com>
+ * gperf support: Giuseppe Scrivano <gscrivan@redhat.com>
+ */
+
+/*
+ * This library is free software; you can redistribute it and/or modify it
+ * under the terms of version 2.1 of the GNU Lesser General Public License as
+ * published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, see <http://www.gnu.org/licenses>.
+ */
+#include <seccomp.h>
+#include <string.h>
+#include "arch.h"
+#include "arch-x86.h"
+%}
+struct arch_syscall_def_internal { int name; int num; int index; };
```

Comment:
I'm not a gperf expert, but I'm assuming that gperf couldn't build a table of `struct arch_syscall_def` like before.  That would remove the need for the weird `static` trickery being done in `x86_64_syscall_iterate()`.  Is my (mis)understanding correct?

Reply:
yeah unfortunately gperf doesn't build the same struct table.  The main issue is that name now is a `int` and we need `stringpool + wordlist[N].name` to get  a pointer back